index.bs

```diff
@@ -391,6 +470,76 @@ interface mixin WebTransport {
         {{WebTransportCloseInfo/errorCode}} and its reason value set to the
         value of {{WebTransportCloseInfo/reason}}.
 
+: <dfn for="WebTransport" method>getStats()</dfn>
+:: Gathers stats for this {{WebTransport}}'s QUIC
+   connection and reports the result asynchronously.</p>
+
+   When close is called, the user agent MUST run the following steps:
+     1. Let |transport| be the WebTransport on which `getStats` is invoked.
+     1. Let |p| be a new promise.
+     1. Return |p| and continue the following steps in background.
+         1. Gather the stats from the underlying QUIC connection.
+         1. Once stats have been gathered, resolve |p| with the
+            {{WebTransportStats}} object, representing the gathered stats.
+
+## Configuration ##  {#web-transport-configuration}
+
+<pre class="idl">
+dictionary WebTransportOptions {
+  sequence&lt;RTCDtlsFingerprint&gt; serverCertificateFingerprints;
+};
+</pre>
+
+<dfn dictionary>WebTransportOptions</dfn> is a dictionary of parameters
+that determine how WebTransport connection is established and used.
+
+: <dfn for="WebTransportOptions" dict-member>serverCertificateFingerprints</dfn>
+:: This option is only supported for transports using dedicated TLS
+   connections, such as `quic-transport`.  For transport protocols that do not
+   support this feature, having this field non-empty SHALL result in a
+   {{NotSupportedError}} exception being thrown.
+:: If supported and non-empty, the user agent SHALL deem a server certificate
+   trusted if and only if it can successfully [=verify a certificate
+   fingerprint=] against {{WebTransportOptions/serverCertificateFingerprints}}
+   and satisfies [=custom certificate requirements=].  The user agent SHALL
+   ignore any fingerprint that uses an unknown {{RTCDtlsFingerprint/algorithm}}
+   or has a malformed {{RTCDtlsFingerprint/value}}.  If empty, the user agent
+   SHALL use certificate verification procedures it would use for normal
+   [=fetch=] operations.
+
+<div algorithm="compute a certificate fingerprint">
+To <dfn>compute a certificate fingerprint</dfn>, do the following:
+1. Let |cert| be the input certificate, represented as a DER encoding of
+   Certificate message defined in [[!RFC5280]].
+1. Compute the SHA-256 hash of |cert|.  Format it as `fingerprint` BNF
+   rule described in Section 5 of [[!RFC8122]].
+
+</div>
+
+<div algorithm="verify a certificate fingerprint">
+To <dfn>verify a certificate fingerprint</dfn>, do the following:
+1. Let |fprs| be the input array of fingerprints.
+1. Let |ref_fpr| be the [=compute a certificate fingerprint|computed
```

Comment:
`fprs` and `ref_fpr` seem to be unnecessarily abbreviated. How about `fingerprints` and `referenceFingerprint`?

Reply:
Done.